parquet-hadoop/src/main/java/org/apache/parquet/hadoop/codec/SnappyCodec.java

```diff
@@ -62,6 +58,10 @@ public Decompressor createDecompressor() {
     return new SnappyDecompressor();
   }
 
+  public DirectDecompressor createDirectDecompressor() {
+    return new SnappyDecompressor.SnappyDirectDecompressor();
+  }
+
```

Comment:
It is now that I have moved the DirectDecompressorPool over from the Drill source.